tests/benchdnn/inputs/conv/test_conv_gpu_ci

```diff
@@ -1,83 +1,68 @@
 --reset
 
---mb=2,16
 --stag=any,axb
---dtag=any,axb
```

Comment:
When do we check `--dtag=axb` now? It looks like we are always testing it with the default value of `any`.

> Dropped --dtag - there is no much value in testing mixed combinations (like --stag=any --dtag=axb). Usually we have both axb or both any

While generally true, mixed layouts is fairly important use case to avoid reorders for interoperation with frameworks. It seems like we should give it some testing at a minimum.

Reply:
`--stag=axb --dtag=any` is \*typically\* resolved to `--stag=axb --dtag=axb` by the implementation (however strictly speaking not always as there are some nuances with 1st convolution). But for clarity maybe better to have it set explicitly like:

```
--stag=axb
--dtag=axb
--batch=option_set_gpu_ci

--stag=any
--dtag=any
--batch=option_set_gpu_ci
```

> While generally true, mixed layouts is fairly important use case to avoid reorders for interoperation with frameworks. It seems like we should give it some testing at a minimum.

Good point. This is usually first/last layers - when the framework needs NHWC/NCHW specifically while keeping output/input as `any`.  I'll add limited coverage similar to the scheme above.

Reply:
Updated in the last version.